Tizanidine/Tizanidine-Report.md

```diff
@@ -0,0 +1,340 @@
+
+**Backman 2008** Backman JT, Schröder MT, Neuvonen PJ. Effects of gender and moderate smoking on the pharmacokinetics and effects of the CYP1A2 substrate tizanidine. Eur J Clin Pharmacol. 2008;64(1):17-24. 
+
+**DrugBank DB00697** (https://www.drugbank.ca/drugs/DB00697)
+
+**Granfors 2004** Granfors MT, Backman JT, Laitila J, Neuvonen PJ. Tizanidine is mainly metabolized by cytochrome P450 1A2 in vitro: Tizanidine is mainly metabolized by cytochrome P450 IA2 in vitro. British Journal of Clinical Pharmacology. 2004 Jan 8;57(3):349–53.
+
+**Haezlewood 1983** Heazlewood V, Symoniw P, Maruff P, Eadie M. Tizanidine- Initial pharmacokinetic studies in patients with spasticity. Eur J Clin Pharmacol. 1983; 25:65-67.
+
+**Henney 2007** Henney HR, Shah J. Relative bioavailability of tizanidine 4-mg capsule and tablet formulations after a standardized high-fat meal: A single-dose, randomized, open-label, crossover study in healthy subjects. Clin Ther. 2007;29(4):661-669.
+
+**Henney 2008** Henney HR, Fitzpatrick A, Stewart J, Runyan JD. Relative bioavailability of tizanidine hydrochloride capsule formulation compared with capsule contents administered in applesauce: A single-dose, open-label, randomized, two-way, crossover study in fasted healthy adult subjects. Clinical Therapeutics. 2008 Dec;30(12):2263–71.
+
+**Kuepfer 2016** Kuepfer L, Niederalt C, Wendl T, Schlender JF, Willmann S, Lippert J, Block M, Eissing T, Teutonico D. Applied Concepts in PBPK Modeling: How to Build a PBPK/PD Model.CPT Pharmacometrics Syst Pharmacol. 2016 Oct;5(10):516-531.
+
+**Mathias 1989** Mathias C, Luckitt J, Desai P, Baker H, Masri W El, Hans I. Pharmacodynamics and pharmacokinetics of the oral antispastic agent tizanidine in patients with spinal cord injury. J Rehabil Res. 1989;26(4):9-16.
+
+**Momo 2010** Momo K, Homma M, Osaka Y, Inomata SI, Tanaka M, Kohda Y. Effects of mexiletine, a CYP1A2 inhibitor, on tizanidine pharmacokinetics and pharmacodynamics. J Clin Pharmacol. 2010;50(3):331-337. 
+
+**PK-Sim Ontogeny Database Version 7.3** (https://github.com/Open-Systems-Pharmacology/OSPSuite.Documentation/blob/38cf71b384cfc25cfa0ce4d2f3addfd32757e13b/PK-Sim%20Ontogeny%20Database%20Version%207.3.pdf)
+
+**Schellenberger 1999** Schellenberger MK, Groves L, Shah J, Novack GD. A controlled pharmacokinetic evaluation of tizanidine and baclofen at steady state. Drug Metabolism and Disposition.1999;2:201–4.
+
+**Shah 2006** Shah J, Wesnes KA, Kovelesky RA, Henney HR. Effects of food on the single-dose pharmacokinetics/pharmacodynamics of tizanidine capsules and tablets in healthy volunteers. Clin Ther. 2006;28(9):1308-1317. 
+
+**SmPC tizanidine** Zanaflex prescribing information. Website: accessdata.fda.gov/drugsatfda_docs/label/2006/020397s021,021447s002lbl.pdf, 2006, Acorda Therapeutics Inc
```

Comment:
Link not properly rendered. Use the full link (http://...

Reply:
fixed